Lib/pickle.py

```diff
@@ -1909,20 +1909,17 @@ def _loads(s, /, *, fix_imports=True, encoding="ASCII", errors="strict",
 
 if __name__ == "__main__":
     import argparse
+    import pprint
     parser = argparse.ArgumentParser(
         description='display contents of the pickle files')
     parser.add_argument(
         'pickle_file',
-        nargs='*', help='the pickle file')
+        nargs='+', help='the pickle file')
     args = parser.parse_args()
-    if not args.pickle_file:
-        parser.print_help()
-    else:
-        import pprint
-        for fn in args.pickle_file:
-            if fn == '-':
-                obj = load(sys.stdin.buffer)
-            else:
-                with open(fn, 'rb') as f:
-                    obj = load(f)
-            pprint.pprint(obj)
+    for fn in args.pickle_file:
+        if fn == '-':
+            obj = load(sys.stdin.buffer)
+        else:
+            with open(fn, 'rb') as f:
+                obj = load(f)
+        pprint.pp(obj)
```

Comment:
```suggestion
        pprint.pprint(obj)
```

Reply:
thank you. I don't know what's happing but I already send my commit after you remind.
Looks like github not showing this changes
![Screenshot from 2025-03-14 16-34-21](https://github.com/user-attachments/assets/a518e702-b7b2-4e6b-bd11-939fb234b2c8)

Reply:
Hmm, strange. Sorry for the back and forth!

Reply:
thank you for review